content/news/2025-XX-XX-bevy-0.16/index.md

```diff
@@ -29,8 +32,14 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 The features above may be great, but what else does Bevy have in flight?
 Peering deep into the mists of time (predictions are _extra_ hard when your team is almost all volunteers!), we can see some exciting work taking shape:
 
-- TODO
+- **A revamped observers API:** Observers are incredibly popular, but come with some weird quirks. Why is `OnAdd` not generic? What does that `B: Bundle` generic even _do_? Please, I want to add observers in my entity spawning to write little callbacks!
```

Comment:
> Please, I want to add observers in my entity spawning to write little callbacks!

I don't understand what this part is saying, needs a little more context imo.

Reply:
```suggestion
- **A revamped observers API:** Observers are incredibly popular, but come with some weird quirks. We're looking to smooth those out, and make them the easiest way to write one-off logic for UI.
```